### ISSUE TITLE
[RUBY] Inconsistent Naming in bigquery data transfer

### ISSUE THREAD

**Issue by TheRoyalTnetennba** [googleapis/gapic-generator#2179]
Folder and file names seem to alternate between 'datatransfer' and 'data_transfer'. 

Readme product page link is dead. I think "https://cloud.google.com/bigquerydatatransfer" should be "https://cloud.google.com/bigquery/transfer/".

Also suffers from https://github.com/googleapis/gapic-generator/issues/2174.

PR: https://github.com/GoogleCloudPlatform/google-cloud-ruby/pull/2220

**Comment by jbolinger** [comment]
Protobuf added a `ruby_namespace` option as of 3.6, which is now in the generator. This should address the issue with the `_pb` files. The proto files will need to be updated though. I just checked and it looks like they don't have the `ruby_namespace` option set (unless you are using a private repo?).

The doc links may be a separate issue that we have to fix. Let's take another look after the protos are updated. This may be the first time we've generated since that change was made.
